spec/models/tenant_spec.rb

```diff
@@ -856,4 +856,21 @@
       expect(projects.map(&:first)).to eq(%w(root.proj3 root.ten1.proj1 root.ten2.proj2))
     end
   end
+  describe "builds tenant tree" do
+    let!(:ten)   { FactoryGirl.create(:tenant) }
+    let!(:ten1)  { FactoryGirl.create(:tenant, :parent => ten) }
+    let!(:ten2)  { FactoryGirl.create(:tenant, :parent => ten1) }
+
+    it "returns subtenants of a tenant" do
+      expect(ten.build_tenant_tree).to eq(
+        [
```

Comment:
At the end of this method call you have `]` and `)` in one line, do the same, i.e. `([` at the beginning.